source/includes/write/gridfs.php

```diff
@@ -0,0 +1,93 @@
+<?php
+require 'vendor/autoload.php'; // include Composer's autoloader
+
+use MongoDB\Client;
+use MongoDB\BSON\ObjectID;
```

Comment:
```suggestion
use MongoDB\BSON\ObjectId;
```

PHP allows case-insensitive class names, but the canonical name is [ObjectId](https://www.php.net/manual/en/class.mongodb-bson-objectid.php).